### ISSUE TITLE
Add documentation and bump version to 2.1.10

### ISSUE THREAD

**Issue by greatfireball** [BioInf-Wuerzburg/SeqFilter#14]
After closing two issues #7 and #12 I think it will be good to add a little documentation about #12 and bump the version number to 2.1.10.

Will you be able to do it @thackl ?

**Comment by thackl** [comment]
Not anytime soon. But feel free to do so if you have the time

**Comment by greatfireball** [comment]
Will try my very best

**Comment by greatfireball** [comment]
Anything special, I need to consider for the documentation? @thackl

**Comment by greatfireball** [comment]
I noticed your comment https://github.com/BioInf-Wuerzburg/SeqFilter/issues/12#issuecomment-462776864 and I found the documentation in https://github.com/BioInf-Wuerzburg/SeqFilter/blob/f907f18b247cef2e5db38999d08374fc5cf6c9f8/bin/SeqFilter#L262-L264

Therefore, I think, it is just the version number change, still waiting. I will prepare an pull request. Additionally, we can consider releases to tag the new versions.
